package.json

```diff
@@ -1,47 +1,154 @@
 {
   "name": "apptentive-react-native",
-  "version": "5.8.3",
```

Comment:
Having 6.0.0 for android, how would this affect Android?

Reply:
This is just the React Native SDK version, which we're moving to 6.0 (in part because we're dropping login/logout support).